x/peggy/keeper/attestation_handler.go

```diff
@@ -74,10 +75,25 @@ func (a AttestationHandler) Handle(ctx sdk.Context, claim types.EthereumClaim) e
 			}
 		}
 
```

Comment:
These events don't look right to me.

1. We should put all attributes in `const` in `types/events.go`.
2. These don't come from message processing, so the `message.*` doesn't make much sense. Instead, perhaps `attestation.*`
3. Use snake_case, e.g. `attestation.send_to_cosmos_amount`

Reply:
Ok, will work on this